site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -34,19 +38,40 @@ An important configuration to note is the PodDisruptionBudget fields, `agones.ex
 
 ## Deployment Considerations
 
-When `SplitControllerAndExtensions` is enabled, what was previously a single `agones-controller` pod is deployed as `agones-controller` and 2 `agones-extensions` pods. For example: 
+{{% feature publishVersion="1.31.0" %}}
+Leader election will automatically be enabled while `SplitControllerAndExtensions` is enabled and while `agones.controller.replicas` is set to > 1. The current default value for [`agones.controller.replicas`]({{< relref "/docs/Installation/Install Agones/helm.md#configuration" >}}) is 2.
+{{% /feature %}}
+
+When `SplitControllerAndExtensions` is enabled, what was previously a single `agones-controller` pod is deployed as {{% feature publishVersion="1.31.0" %}}, possibly 2+,{{% /feature %}} `agones-controller` and 2 `agones-extensions` pods. For example: 
```

Comment:
since you already have a publishVersion block here, just duplicate this paragraph to make it easier on the review rather than embedding a publish in the middle.